src/Estimators/src/SubModel.cpp

```diff
@@ -494,16 +503,100 @@ const RDE::FT& RDE::SubModel::getFTSensor(const int index) const
     return m_ftList.at(index);
 }
 
+const RDE::FT& RDE::SubModel::getFTSensor(const std::string name) const
+{
+    for (int ftIndex = 0; ftIndex < m_ftList.size(); ftIndex++)
```

Comment:
You might be interested in using an ``unordered_map`` . In this way, instead of performing a linear search, you can simply access the FT by name (provided that exists, but you can check it with the ``find`` method). This in general is beneficial when you have a lot of entries, but maybe it is not the case here. Another  difference is that the order is no more preserved. But if you are also interested in the order, you can simply have an additional vector with the associated names in the given order, or with the pointers to corresponding objects.

The same holds below